markdown/dev/howtos/git/editing-on-github/en.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Editing freesewing content on Github
+---
+
+## Introduction
+This 'How to' is designed to help get you started editing content using the [Github website](https://github.com/freesewing) where you can edit pattern instructions and website pages.  For info, it is also possible to make changes using the Git desktop application and/or your computer's command line; these advanced topics are beyond the scope of this 'How to'.
+
+Git allows Freesewing supporters to suggest changes and helps the administrators to review the code and implement approved changes. If you are new to Git, you will need to [create an account](https://github.com/signup?user_email=&source=form-home-signup).
+
+All pattern instructions and web pages (including this 'How to') are written using Markdown code. Please see our [Markdown guide](https://freesewing.dev/guides/markdown) for more information on this.
+
+All the *live* code and images are stored in the Freesewing repository.  For security, most contributors don't have permission to change the 'live' code.  Instead we must follow three steps to make changes:
+
+1. *Fork* (create a copy of) the repository.  This allows you to experiment with changes without affecting the original project.
+2. Make the edits.
+3. Save your edits and submit a *pull request* to let the admins know about your changes.
+
+The Github documentation provides a more [detailed explanation](https://docs.github.com/en/get-started/quickstart/fork-a-repo) of this.
+
+## Fork the repository
+- On Github.com, navigate to the freesewing [repository](https://github.com/freesewing/freesewing)
+- In the top-right corner of the page, click **Fork**
+![Fork button](fork_button.png)
+- It is helpful to give your fork a name to help you and others know what has changed. E.g. if you are going to edit the Simone pattern instructions to include info on using velcro fastenings, you could call your fork "Simone-Velcro".
```

Comment:
Here, you are confusing a fork with a branch, which I'm afraid to say are different things.
You don't name a fork. It will have the same name, but under a different user.
So if you fork `freesewing/freesewing` it will be `joeschofield0/freesewing`.

I also feel that creating a dedicated branch for changes is perhaps a bit much for new users.
There's nothing wrong with making changes in the default branch (`develop`) and then submit a pull request from it.